src/components/common/Chart/Chart.tsx

```diff
@@ -167,7 +167,7 @@ function ChartContent(props: ChartContentProps): JSX.Element {
     };
     createChart();
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [chartLabels, chartValues, locale]);
+  }, [chartData, locale]);
```

Comment:
suggestion: use `useMemo` in all the clients to populate the chartData, they currently set the chartData in the props, which will trigger this useEffect a lot.. you could optionally test whether this is triggered a lot

Reply:
Yeah, I am seeing quite a bit of that effect running. I'll look into your suggestion. Thanks!